common/scaffolding/reports/ally-assess-1.json

```
@@ -0,0 +1,635 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+  "@type": [
+    "earl:Assertion",
+    "acr:AssessmentReport"
+  ],
+  "dct:date": "2024-02-29",
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": [
+      "https://atlakm.github.io/GCWeb/common/scaffolding/text-level-semantics.html"
+    ],
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Text level semantic element in the main page area",
+    "oa:hasState": [
+      {
+        "@type": "acr:MediaQueryList",
+        "acr:stateValue": "(min-width: 992px) and (max-width: 1199px)",
+        "_:iterator": 0
+      }
+    ],
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "main"
+    }
+  },
+  "dct:description": "1. Some scaffolding components do not convey meaning semantically and may need styling to make apparent. For example, quotations for the &lt;cite>. element.\n2. The landmark error from axe report below is from the master template and unrelated to this test and can be ignored. Marking as passed.\n3. Note that some tags like &lt;b>, &lt;i>, and &lt;u> used to be decorative but have been decided to be semantic and so don't fail SC1.3.1.",
+  "earl:result": [
+    {
+      "earl:test": "WCAG21:non-text-content",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:passed"
+    },
+    {
+      "earl:test": "WCAG21:audio-only-and-video-only-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+    },
+    {
+      "earl:test": "WCAG21:captions-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+    },
```

There is still a few JSON validation error and the through the file.

```suggestion
      "earl:outcome": "earl:inapplicable"
    },
    {
      "earl:test": "WCAG21:captions-prerecorded",
      "earl:subject": "_:subject",
      "earl:result": [],
      "acr:asset": [],
      "earl:mode": "earl:automatic",
      "acr:severity": "acr:noSeverity",
      "acr:relevancy": "wbv:acr#noRelevancy",
      "dct:description": "",
      "earl:outcome": "earl:inapplicable"
    },
```